webapp/channels/src/components/post_view/gm_conversion_message/gm_conversion_message.tsx

```diff
@@ -36,8 +36,9 @@ function GMConversionMessage(props: Props): JSX.Element {
         ),
     );
 
-    const convertedByUserUsername = userProfiles.find((user) => user.id === convertedByUserId)!.username;
-    const gmMembersUsernames = userProfiles.map((user) => renderUsername(user.username));
+    // we can memoize on profile length as the number of users in a GM always stay constant
+    const convertedByUserUsername = useMemo(() => userProfiles.find((user) => user.id === convertedByUserId)!.username, [userProfiles.length]);
```

Comment:
0/5 If we're memoizing these, should we also memoize the calls to `renderUsername` and `intl.formatList`?

Reply:
Done.

Reply:
Removed momomization from here as discussed.